### DISCUSSION TITLE
Aggregated statistics

### DISCUSSION THREAD

**Discussion by julianrutten** [postalserver/postal#2096] [Feature suggestions]
Hi there,

It would be very helpful for e-mail (template) analysis if we can aggregate data for example on tag and be able to view open and click rates for those aggregates. I searched the issues list here and the docs but find no mention of it.

Awesome app!

Kind regards,

Julian Rutten

**Comment by julianrutten** [comment]
Okay so my bad there are a lot of requests for this already.

**Comment by willpower232** [comment]
Postal isn't designed to hold on to messages for a long time (or group messages by any particular category) so you'd definitely do much better to implement webhooks and record statistics in your own application.
